docsrc/content/abstraction-semigroup.fsx

```diff
@@ -42,6 +42,7 @@ From .Net/F#
  -  ``Map<'T,'U>``
  -  ``TimeSpan`` 
  -  ``Tuple<*>``
+ -  ``ValueTuple<*>``
```

Comment:
This is tricky. either we support `ValueTuple<n>` with `n < 8` or we do some kind of reflection trick for those tuples greater than 8.

Reply:
we support for n > 0 and n <9

Reply:
Should I remove this line?

Reply:
No, in that case just change it to `ValueTuple<*> (* up to 8 elements)`